libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1400,6 +1403,11 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator displayConfirmationMe
     [[SFRootViewManager sharedManager] pushViewController:alert];
 }
 
+- (void)oauthCoordinatorDidCancelBrowserAuthentication:(SFOAuthCoordinator*)coordinator
+{
+    [self delegateDidCancelBrowserFlow];
```

Comment:
Hitting done when the safari view controller is shown and hitting cancel on the alert that precedes safari being launched should have the same effect.